translations/README.md

```diff
@@ -2,7 +2,7 @@
 
 | File            | Language      |
 |-----------------|---------------|
-| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md)                   ￨ [🇬🇷 Ελληνικά](README.greek.md)  |
+| Readme          | [:uk: English](../README.md)          ￨ [:fr: Français](README.french.md)          ￨ [:jp: 日本語](README.japanese.md)                   ￨ [🇬🇷 Ελληνικά](README.greek.md)  | [:indonesia: Indonesian](README.indonesian.md)  | 
```

Comment:
This doesn't show up properly here: 

![image](https://user-images.githubusercontent.com/5722022/68850560-40faf280-06fa-11ea-92b7-2f326b58143c.png)

Reply:
Hi @marionschleifer nice catch, thanks!

Apparently the pipe character is conflicting with the table markup. I'll fix it quickly.

Reply:
it's fixed with the next commit: https://github.com/hasura/graphql-engine/pull/3258/commits/8ccc51b9564c977f516ae358a62d5138d23077a2